src/sentry/feedback/endpoints/organization_feedback_summary.py

```diff
@@ -141,3 +157,30 @@ def get(self, request: Request, organization: Organization) -> Response:
                 "numFeedbacksUsed": len(group_feedbacks),
             }
         )
+
+
+def make_seer_request(request: SummaryRequest) -> bytes:
+    serialized_request = json.dumps(request)
+
+    response = requests.post(
+        f"{settings.SEER_AUTOFIX_URL}/v1/automation/summarize/feedback/summarize",
+        data=serialized_request,
+        headers={
+            "content-type": "application/json;charset=utf-8",
+            **sign_with_seer_secret(serialized_request.encode()),
+        },
+    )
+
+    if response.status_code != 200:
+        logger.warning(
```

Comment:
```suggestion
        logger.error(
```
would error make more sense here?